fv3core/pace/fv3core/initialization/analytic_init.py

```diff
@@ -0,0 +1,64 @@
+from enum import Enum
+
+import pace.util as fv3util
+from pace.fv3core.dycore_state import DycoreState
+from pace.util.grid import GridData
+
+
+class cases(Enum):
+    baroclinic = "baroclinic"
+    tropicalcylclone = "tropicalcyclone"
+
+
+valid_cases = [item.value for item in cases]
+
+
+def init_analytic_choice(
+    analytic_init_str: str,
+    grid_data: GridData,
+    quantity_factory: fv3util.QuantityFactory,
+    adiabatic: bool,
+    hydrostatic: bool,
+    moist_phys: bool,
+    comm: fv3util.CubedSphereCommunicator,
```

Comment:
Would `init_analytic_state` be a better name for the function? Relatedly I think we could just use `case` instead of `analytic_init_str`

Also it looks like the adiabatic, hydrostatic, and moist_phys args aren't actually passed on to the subsequent function calls --should we remove them for the call signature?

Reply:
Or we should pass the values into the functions